darwin/exporter/formats/nifti.py

```diff
@@ -489,18 +495,10 @@ def unnest_dict_to_list(d: Dict) -> List:
     volumes = unnest_dict_to_list(output_volumes)
     for volume in volumes:
         img = nib.Nifti1Image(
-            dataobj=np.flip(volume.pixel_array, (0, 1, 2)).astype(np.int16),
+            dataobj=volume.pixel_array.astype(np.int16),
             affine=volume.affine,
         )
```

Comment:
Until this PR, this line was flipping the orientation of every axis in every NifTI export